### PR TITLE
setup.py: Add installation

### PR DESCRIPTION
**Pull request by Redridge**
Also modify requirements.txt and make minor
adjustments to the file structure

### CODE REVIEW

File: scripts/coala_quickstart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python3
+
+import sys
+
+def main():
+  files_glob = input("Please enter the files glob: ")
+  coafile = open(".coafile", "a+")
```
Comment (sils1297) [inline scripts/coala_quickstart]: indentation's wrong btw.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          platforms='any',
+          packages=find_packages(exclude=["build.*", "*.tests.*", "*.tests"]),
+          install_requires=required,
+          license="AGPL-3.0",
+          long_description="coala-quickstart is a tool to help you "
+                           "set up coala easier "
+                           "coala - the COde AnaLysis Application.",
+          entry_points={ 
+              "console_scripts": [ 
+                  "coala-quickstart = scripts.coala_quickstart:main" ]},
```
Comment (sils1297) [inline setup.py]: so IIUC this doesn't work?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+with open('requirements.txt') as requirements:
+    required = requirements.read().splitlines()
+
+if __name__ == "__main__":
+    setup(name='coala-quickstart',
+          version='0.1.0.dev0',
+          description='Quickstart for coala (Code Analysis Application)',
+          author="The coala developers",
+          maintainer="Lasse Schuirmann, Fabian Neuschmidt, Mischa Kr\xfcger",
```
Comment (sils1297) [inline setup.py]: nope, maintainers are @Redridge and @Adrianzatreanu  I guess

Comment (sils1297) [inline setup.py]: if you don't mind ;)

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+              "console_scripts": [ 
+                  "coala-quickstart = scripts.coala_quickstart:main" ]},
+          # from http://pypi.python.org/pypi?%3Aaction=list_classifiers
+          classifiers=[
+              'Development Status :: 4 - Beta',
+
+              'Environment :: Console',
+              'Environment :: MacOS X',
+              'Environment :: Win32 (MS Windows)',
+              'Environment :: X11 Applications :: Gnome',
```
Comment (sils1297) [inline setup.py]: coala quickstart itself has nothing to do with gnome

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+              'Programming Language :: Python :: Implementation :: CPython',
+              'Programming Language :: Python :: 3.3',
+              'Programming Language :: Python :: 3.4',
+              'Programming Language :: Python :: 3.5',
+              'Programming Language :: Python :: 3 :: Only',
+
+              'Topic :: Scientific/Engineering :: Information Analysis',
+              'Topic :: Software Development :: Quality Assurance',
+              'Topic :: Text Processing :: Linguistic']
+		)
```
Comment (sils1297) [inline setup.py]: indentation is weird, either four spaces less or close bracket in previous line IMO

File: setup.py (hunk 3 above)
Comment (Redridge) [inline setup.py]: ok, I just copied that from the previous setup.py ;)

File: setup.py (hunk 2 above)
Comment (Redridge) [inline setup.py]: It builds but `coala-quickstart` is not in the path

File: scripts/coala_quickstart (hunk 1 above)
Comment (sils1297) [inline scripts/coala_quickstart]: why is this no .py file? For some platforms this is needed, that might be your issue actually

Comment (Redridge) [inline scripts/coala_quickstart]: Ok will change that right away, maybe that is the problem

File: setup.py (hunk 5 above)
Comment (Redridge) [inline setup.py]: I will just run coala over it

File: .coafile [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+[autopep8]
+bears = PEP8Bear
+
+default_actions = PEP8Bear: ApplyPatchAction
+
+[linelength]  # Sometimes autopep8 makes too long lines, need to check after!
+bears = LineLengthBear
+
+[commit]
+bears = GitCommitBear
```
Comment (SanketDG) [inline .coafile]: Newline needed.

Comment (Redridge) [inline .coafile]: Will do

**Comment by sils1297** [comment]
pulled into https://github.com/coala-analyzer/coala-quickstart/pull/9

**Pull request closed without merging**